packages/node_modules/@webex/react-container-message-composer/src/components/ComposerButtons.js

```diff
@@ -2,16 +2,19 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import bowser from 'bowser';
+import uuid from 'uuid';
 
 import {Icon} from '@momentum-ui/react';
 
 import styles from './ComposerButtons.css';
 
 function ComposerButtons({onAttachFile, composerActions}) {
+  const inputId = uuid.v4();
```

Comment:
Could we give this a somewhat readable ID?
```suggestion
  const inputId = `messageFileInput-${uuid.v4()}`;
```